### ISSUE TITLE
question: are you going to be picking agm up?

### ISSUE THREAD

**Issue by kripto202** [MarcBook/AGM#1]
I have to ask this. Are you going to be picking agm up if they let agm go?

**Comment by MarcBook** [comment]
Yes, I want pick it up. I already talked to some German Modders.

**Comment by kripto202** [comment]
ok I'm always willing to throw in some ideas to help progress agm. This mod cannot die because of MANW

**Comment by kripto202** [comment]
I'm also willing to help with some pictures and designs if needed.

**Comment by MarcBook** [comment]
That sounds good, I'm sure we will need some pictures.

**Comment by J-Ziegler** [comment]
If it isn't an April Fools hoax all around, I'll see if I can't teach myself anything useful, maybe give a hand.

**Comment by NicSchni** [comment]
PFFF
just wait for ACE3
Did you know that Koffeinflummi is on their Dev list?
https://dev.withsix.com/projects/show/ace-mod2
Just like Glowbal from CSE

**Comment by kripto202** [comment]
yes I saw that about an hour ago

**Comment by MarcBook** [comment]
KoffeinFlummi said yesterday I should wait 2 or 3 days.

**Comment by MarcBook** [comment]
http://ace3mod.com/
